### PR TITLE
feat: [Week-2.9] Update and Translate "JavaScript Error Handling" Module

### PR DESCRIPTION
**Pull request by KristiBo**
#### 🤔 This is a ...

- [ ] 🌟 New task
- [ ] 🌐 New module
- [ ] ⚙️ Update to an existing task
- [x] 🔧 Update to an existing module
- [x] 🔗 Update or addition of external resources or links
- [ ] 🐛 Fix in a task or related content
- [ ] 🛠 Fix in a module or related content
- [ ] ✏️ Fixed a typo or grammatical error
- [ ] 🔗 Fixed a broken link
- [ ] ❓ Other (specify: **\*\*\*\***\_\_\_\_**\*\*\*\***)

#### Description

- [x] Translate all module content into English
- [x] Review and update the module content

#### Additional Information

Issue: [1523](https://github.com/rolling-scopes-school/tasks/issues/1523)

#### Checklist

- [x] ✅ I have performed a self-review of my own code.
- [ ] 📝 I have commented my code, particularly in hard-to-understand areas.
- [ ] 🔧 I have made corresponding changes to the documentation (if applicable).
- [x] 🚫 My changes generate no new warnings or errors.

### CODE REVIEW

**Review by haslie22 — APPROVED** [review]

**Review by haslie22 — APPROVED** [review]
👍

File: stage2/modules/error-handling/README.md [hunk 1]
```diff
@@ -1,25 +1,44 @@
...
+Students are encouraged to study the following resources:
 
-2 часа
+- RS School Lecture "JS Error Handling" (RU): [video](https://www.youtube.com/watch?v=99Qz8xRbl4k), [slides](https://slides.com/dzmitrytsebruk/error-handling)
+- [Error Object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error)
+- [`throw` Statement](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/throw)
+- [`try...catch` Statement](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch)
+- [Window: error event](https://developer.mozilla.org/en-US/docs/Web/API/Window/error_event)
+- [Custom Errors](https://www.w3docs.com/learn-javascript/custom-errors-extending-error.html)
+- [Error Handling with Promises](https://www.w3docs.com/learn-javascript/error-handling-with-promises.html)
```
Comment (valerydluski) [inline stage2/modules/error-handling/README.md:32]:
```suggestion
- [Error Handling with Promises](https://www.w3docs.com/learn-javascript/error-handling-with-promises.html)
- [Error handling](https://javascript.info/error-handling)
```

File: stage2/modules/error-handling/README.md [hunk 2]
```diff
@@ -1,25 +1,44 @@
...
```
Comment (valerydluski) [inline stage2/modules/error-handling/README.md:33]: Please add the section: Types of Errors

[AggregateError](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/AggregateError)
[EvalError](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/EvalError)
[InternalError](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/InternalError)
[RangeError](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RangeError)
[ReferenceError](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ReferenceError)
[SyntaxError](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SyntaxError)

**Review by valerydluski — COMMENTED** [review]
Good job, thank you.

**Review by valerydluski — APPROVED** [review]

**Pull request merged**